test/unit/libraries/k8s_pod_test.rb

```diff
@@ -0,0 +1,58 @@
+require_relative 'resource_test'
+
+class K8sPodTest < ResourceTest
+  STUB_DATA = {
+    'v1': {
+      default: {
+        pods: [
+          {
+            name: 'pod1',
+            kind: 'pod',
+            status: {
+              phase: 'running'
+            },
+            metadata: {
+              uid: 'abcd1234',
+              name: 'pod1',
+              namespace: 'default',
+              resourceVersion: 1234,
+              annotations: [],
```

Comment:
Annotations and Labels are k8s resource object with key value and not a array. May be we need  to have a look at this.